### PR TITLE
Added a mechanism to get the current block numbers.

### PR DESCRIPTION
**Pull request by Andrew-Pohl**
Grab the latest block numbers for Fuse and Eth and update the .env files with the latest blocks.

### CODE REVIEW

File: scripts/quickstart.sh [hunk 1]
```diff
@@ -24,6 +26,8 @@ ADDRESS=""
 NODE_KEY=""
 INSTANCE_NAME=""
 PLATFORM=""
+ETHBLOCKNUMBER=""
+FUSEBLOCKNUMBER=""
```
Comment (Andrew-Pohl) [inline scripts/quickstart.sh:30]: Actually didn't need these globals all done inside the new functions. I will delete them in the morning :)

**Comment by Andrew-Pohl** [comment]
Also wasn't sure if the bridge-data folder should be removed from the script to. Can add it of needed!

**Comment by LiorRabin** [comment]
> Also wasn't sure if the bridge-data folder should be removed from the script to. Can add it if needed!

Not needed at this point, it is only needed as an error recovery. 
If the blocks are updated properly than any new nodes will start monitoring from the proper blocks and won't encounter the errors we had :)

File: scripts/quickstart.sh (hunk 1 above)
Comment (LiorRabin) [reply scripts/quickstart.sh:30]: Cool, so remove them when you can and I'll merge the PR

Comment (Andrew-Pohl) [reply scripts/quickstart.sh]: Done :)

**Pull request merged**